server.py

```diff
@@ -16,42 +19,55 @@ def setup():
         socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
     )
     sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-    sock.bind(addr)
-    sock.listen(2)
+    sock.bind(ADDR)
+    sock.listen(4)
     return sock
 
 
-def response_ok():
-    """Return Header and Body information for Response 200."""
-    response_headers = (
-        b'HTTP/1.1 200 OK\r\n'
-        + date +
-        b'Content-Type: text/html\r\n'
-        b'Content-Length:\r\n')
+def response_ok(_body, _type):
 
-    response_body = (
-        b'<html><body>'
-        b'<p>All good here, captain.</p>'
-        b'</body></html>')
+    _date = email.Utils.formatdate(usegmt=True)
 
-    return response_headers + response_body
+    _RESPONSE_TEMPLATE = _CRLF.join([
+        b"HTTP/1.1 200 OK",
+        b"{date}",
+        b"Content-Type: {content_type}",
+        b"Content-Length: {content_length}",
+        b"",
+        b"{content_body}",
+        b"",
+    ])
```

Comment:
It's not a super big deal, but if this were a module-level constant instead of something defined *inside* this function, it would only need to be constructed once, instead of each time the function is called.